### PR TITLE
[s] Lowers xenomorph acid damage on living mobs

### LINKED ISSUE #12901: Alien corrosive acid one shots borgs, most simple mobs... and people.
**Issue Description**:

Alien corrosive acid one shots most simple mobs, borgs, and even people, dealing 200 damage to simple mobs, or 140 brute and 280 burn to humanoids instantly

**What did you expect to happen**: 

It to start slowly burning the target, needing a shower or fast repairs to keep them alive, but not too much of an issue at the exact moment.

**What happened instead**:

It instantly kills anything short of megafauna, by dealing 200 burn, or 140 brute and 280 burn to humanoids.

**Why is this bad/What are the consequences**:

Xenos already have enough balance issues, instantly killing anything buy standing beside them is a bit much 

**Steps to reproduce the problem**:

Be a xeno sentinel / queen / xeno organ user.
Right click on target
Press corrosive acid
Watch as the person instantly goes hard crit from the damage, and dies seconds later.

**When did the problem start happening**:

Object damage rework changed the way the acid worked, and would have changed this.

**Extra information**:

https://imgur.com/a/FxnZrP5

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Title. Xenomorph acid now does less damage to living mobs.
In return, any acid on the chest (As far as I know, only xenomorphs and probably acid foam/acid smoke can actually touch this) has doubled strength and is applied twice. This means that the maximum acid damage is doubled, and it will reach the maximum damage faster.
Naked humans take 120 burn damage and 60 brute damage from this acid.
Humans with no headgear take 40 burn damage and 20 brute damage.
Fixes #12901 

## Why It's Good For The Game
Fixes a bug and makes coming into the melee range of xenos slightly more interesting and fair. 
Some other testing:
- Skrell in security officer outfit: jumpsuit & protective vest melted in about 25 seconds, helmet and others in about 30 seconds
- Skrell in HOS outfit: same numbers for the helmet and others, armored jacket didn't seem to melt due to high acid armor
- Any MODsuit seems to be immune to this acidmelting of xenos

## Testing
See the section in `Why it's good for the game`

## Changelog
:cl:
tweak: Acid melts (jump)suits twice as fast now
tweak: xenomorph human acid damage has been reduced
/:cl:

### CODE REVIEW

File: code/datums/spells/alien_spells/corrosive_acid_spit.dm [hunk 1]
```diff
@@ -20,6 +20,11 @@
 	if(!plasma_check(200, user))
 		to_chat(user, "<span class='noticealien'>You don't have enough plasma to perform this action!</span>")
 		return
+	if(isliving(target) && target.acid_act(100, 50)) // If we don't do this, naked humans/simplemobs get absolutely nuked
```
Comment (Contrabang) [inline code/datums/spells/alien_spells/corrosive_acid_spit.dm:23]: Can you please make this a variable and just change it if the target is living, to reduce duplicate code?

Comment (DGamerL) [reply code/datums/spells/alien_spells/corrosive_acid_spit.dm:23]: I'm sorry but I've been looking at it and don't quite get what you mean, could you give some sort of example?

File: code/datums/spells/alien_spells/corrosive_acid_spit.dm [hunk 2]
```diff
@@ -20,7 +20,10 @@
 	if(!plasma_check(200, user))
 		to_chat(user, "<span class='noticealien'>You don't have enough plasma to perform this action!</span>")
 		return
-	if(target.acid_act(200, 100))
+	var/acid_damage_modifier = 2
+	if(isliving(target))
+		acid_damage_modifier = 1
+	if(target.acid_act(100 * acid_damage_modifier, 50 * acid_damage_modifier))
```
Comment (Contrabang) [inline code/datums/spells/alien_spells/corrosive_acid_spit.dm:26]:
```suggestion
	var/acid_damage_modifier = 100
	if(isliving(target))
		acid_damage_modifier = 50
	if(target.acid_act(2 * acid_damage_modifier, acid_damage_modifier))
```

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 3]
```diff
@@ -323,7 +323,7 @@ emp_act
 			chest_clothes = wear_suit
 		if(chest_clothes)
 			if(!(chest_clothes.resistance_flags & UNACIDABLE))
-				chest_clothes.acid_act(acidpwr, acid_volume)
+				chest_clothes.acid_act(2 * acidpwr, 2 * acid_volume)
```
Comment (Contrabang) [inline code/modules/mob/living/carbon/human/human_defense.dm:326]:
```suggestion
				chest_clothes.acid_act(acidpwr, acid_volume)
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**